setup.cfg

```diff
@@ -218,7 +218,18 @@ check_untyped_defs = False
 [mypy-test.functional.webhooks.tour_generator]
 check_untyped_defs = False
 [mypy-galaxy.util.compression_utils]
-check_untyped_defs = False
+disallow_any_generics = True
+disallow_subclassing_any = True
+disallow_untyped_calls = False
+disallow_untyped_defs = True
+disallow_incomplete_defs = True
+check_untyped_defs = True
+disallow_untyped_decorators = True
+no_implicit_optional = True
+warn_unused_ignores = True
+warn_return_any = True
```

Comment:
If `warn_return_any = True` is specified under `[mypy-galaxy.tools.wrappers]` then `tox -e test_galaxy_packages` fails

https://app.circleci.com/pipelines/github/galaxyproject/galaxy/19036/workflows/9cc3867a-c0f9-4d2e-8876-9a49ed4e54a6/jobs/156058?invite=true#step-104-2353

```
mypy galaxy tests
galaxy/tools/wrappers.py:262: error: Returning Any from function declared to return "Iterator[Any]"
galaxy/tools/wrappers.py:265: error: Returning Any from function declared to return "bool"
galaxy/tools/wrappers.py:346: error: Returning Any from function declared to return "List[Tuple[str, str]]"
galaxy/tools/wrappers.py:365: error: Returning Any from function declared to return "bool"
galaxy/tools/wrappers.py:573: error: Returning Any from function declared to return "List[List[Any]]"
```

but `tox -e py36-mypy` passes